src/SayMore/UI/ComponentEditors/ContributorsEditor.cs

```diff
@@ -48,6 +55,63 @@ public partial class ContributorsEditor : EditorBase
 				personInformant.PersonUiIdChanged += HandlePersonsUiIdChanged;
 		}
 
+		private void AddControls(string imageKey)
```

Comment:
So imageKey is used like a boolean in this method. This is confusing. Maybe rename the method AddSessionControls (if that's accurate) and have only the part inside the if. Then put the if/else logic where the method is called with a comment explaining why imageKey means we need to add different controls?